IBX-7978: Added app switcher info (#297)
ibexa/documentation-user: docs/getting_started/discover_ui.md

- Global search field that allows you to [find content](../search/search_for_content.md) by checking all searchable Fields
- Drop-down that changes the site context
- Notification icon that informs you about incoming assignments, for example, items assigned for [your review](../content_management/workflow_management/editorial_workflow.md#review-queue)

> - App switcher which allows to quickly pivot between applications within the QNTM group ecosystem

Reviewer: ```suggestion
- App switcher which allows to quickly switch between applications within the QNTM group ecosystem
```
To pivot is to turn or rotate, like a hinge. Toggle also woudn't do because you have multiple options.

> - App switcher which allows to quickly switch between applications within the QNTM group ecosystem

Reviewer 2: @juskora in practice and in the current state, "app" switcher includes links to QNTM companies official websites (not links to backoffice apps). I would change this sentence a bit e.g.:
_..which includes/presents the links to official websites of companies - world-class ad-tech and martech solutions providers being part of QNTM ecosystem._
or in other form:
__"get to know other world-class mar-tech and ad-tech solutions offered by companies being part of QNTM ecosystem"_

> - App switcher which includes links to official websites of the QNTM group ecosystem companies, world-class ad-tech and mar-tech solution providers

Reviewer: ```suggestion
- App switcher that includes links to official websites of the QNTM group companies, including world-class ad-tech and mar-tech solution providers
```

Reviewer 2: Can we use different verb in the second part, because currently we repeat "includes"... "including" :)

Author: Let's leave it with the first version, without this suggestion :)
